src/common/system/testdata/sample_host_ubuntu/proc/version_signature

```diff
@@ -0,0 +1 @@
+Ubuntu 5.4.0-52.57-generic 5.4.65
```

Comment:
I believe the `src/stirling/utils/testdata/sample_host_{ubuntu,debian}` directories can be removed.